std_msgs/String LookupWord
---
std_msgs/String Explanation
